src/app/api/tasks/tasks.service.ts

```diff
@@ -332,6 +332,7 @@ export class TasksService extends BaseService {
       where: { assigneeId, assigneeType, workspaceId: this.user.workspaceId },
     })
     await this.db.label.deleteMany({ where: { label: { in: labels } } })
+    return tasks
```

Comment:
just curious, what is `db.label`? why do we need to delete that?

Reply:
As for deleting it, I'm not very familiar with the implementation, and the implementation for `label` seems to be a bit awkward... I think it was Ananta or Arpan who implemented this the last time around - and I didn't want to remove this in case deleting the corresponding label was necessary.